packages/lodestar/src/tasks/index.ts

```diff
@@ -82,7 +93,10 @@ export class TasksService implements IService {
         finalized
       ).run();
       // should be after ArchiveBlocksTask to handle restart cleanly
-      await new ArchiveStatesTask(this.config, {db: this.db, logger: this.logger}, finalized).run();
+      const lastStoredEpoch = (await this.db.stateArchive.lastKey()) as number;
```

Comment:
we changed from storing by epoch to by slot https://github.com/ChainSafe/lodestar/blob/master/packages/lodestar/src/db/api/beacon/repositories/stateArchive.ts#L17